azure-functions/acbs-function/api.js

```diff
@@ -56,13 +62,40 @@ const get = async (endpoint) => {
  */
 const post = async (endpoint, payload) => {
   if (endpoint && payload) {
-    const url = `${process.env.APIM_MDM_URL}/${endpoint}`;
+    const url = `${process.env.APIM_TFS_URL}/${endpoint}`;
 
     return axios({
       method: 'post',
       url,
+      headers: tfs,
+      data: [payload],
+    }).catch((error) => {
+      console.error('Error calling ACBS POST /%s', endpoint);
+
+      return {
+        status: error.response ? error.response.status : error,
+        data: { error: error.response ? error.response.data : error },
+      };
+    });
+  }
+
+  return badRequest;
+};
 
-      headers,
+/**
+ * Invokes MDM POST endpoint
+ * @param {String} endpoint MDM endpoint
+ * @param {Object} payload Payload object
+ * @returns {Object} API response object
+ */
+const postMdm = async (endpoint, payload) => {
+  if (endpoint && payload) {
+    const url = `${process.env.APIM_MDM_URL}/${endpoint}`;
+
+    return axios({
+      method: 'post',
+      url,
+      headers: mdm,
       data: [payload],
     }).catch((error) => {
       console.error('Error calling ACBS POST /%s', endpoint);
```

Comment:
As this is the post to MDM, it might be worth changing the error message to `Error calling MDM POST /%s` instead?

Reply:
Good shout, have re-written error messages.

Reply:
Thanks, happy to resolve this